src/mqtt.cpp

```diff
@@ -151,9 +151,18 @@ void sendHAConfig()
   const bool retain = true;
   const int qos = 1;
   char buff[512];
-  const int capacity = JSON_OBJECT_SIZE(10); // intentionally larger than we need.
+  const int capacity = JSON_OBJECT_SIZE(24); // intentionally larger than we need.
   StaticJsonDocument<capacity> doc;
 
+  // deviceinfo
+  const int devcapacity = JSON_OBJECT_SIZE(6);
+  StaticJsonDocument<devcapacity> deviceinfo;
+  deviceinfo.clear();
+  deviceinfo["manufacturer"] = "e-radionica";
+  deviceinfo["model"] = "Inkplate 10";
+  deviceinfo["name"] = "HomePlate";
```

Comment:
Should this be the hostname? (which defaults to"homeplate")

Reply:
Easiest would be hostname. Extra friendly name seems to be overkill and name can be easily changed in HA.